bundle/Dockerfile

```diff
@@ -82,6 +87,8 @@ COPY --from=build /go/bin/certificate-authority /usr/local/bin/certificate-autho
 COPY --from=cloud-build /go/src/github.com/plgd-dev/cloud/certificate-authority/config.yaml /configs/certificate-authority.yaml
 COPY --from=build /go/bin/certificate-generator /usr/local/bin/certificate-generator
 COPY --from=build /go/bin/nats-server /usr/local/bin/nats-server
+COPY --from=build /go/bin/nats /usr/local/bin/nats
+COPY --from=cloud-build /go/src/github.com/plgd-dev/cloud/bundle/jetstream.json /configs/jetstream.json
```

Comment:
plgd service configuration is exposed using environment variables. In the case of the jetstream, it's only this file. Users might be interested in updating it by mounting a volume, but if they do it, they remove all service configuration files. Add it one level deeper, /configs/jetstream/jetstram.json so in case of users want to change it, they will map just this volume containing the jetstream configuration.

Reply:
Docker allows you to mount file instead of directory

Reply:
https://stackoverflow.com/questions/42248198/how-to-mount-a-single-file-in-a-volume